modules/project-comments/server/routes/comment.routes.js

```diff
@@ -149,6 +149,57 @@ module.exports = function (app) {
 			return model.getCommentsForPeriod (periodId, eaoStatus, proponentStatus, isPublished, commentId, authorComment, location, pillar, topic, skip, limit, sortby);
 		}));
 
+	app.route ('/api/comments/period/:periodId/perms/sync').all(policy ('guest'))
+		.put (routes.setAndRun (CommentModel, function (model, req) {
+
```

Comment:
This whole block of code doesn't belong in the routes.js file, it should be a named function inside the associated controller.js file.